vendor/patches/lowrisc_ip/rv_plic/0001-Remove-Alerts.patch

```diff
@@ -1,122 +0,0 @@
-diff --git a/data/rv_plic.hjson.tpl b/data/rv_plic.hjson.tpl
```

Comment:
Don't we still need this? As part of this PR we should re-generate the PLIC with the updated templates.

Reply:
Rebased and regenerated PLIC now; I've opted to leave reggen/templates such that it includes the `OpenTitan project` reference and so this is present in the generated PLIC, but left the Sonata-specific RTL (which I haven't needed to update) without reference in the copyright headers; this does mean - for now - that the output from util/reg_gen.sh mismatches trivially, but it sounds as though there are other twiddles to be done also at a later date.